apps/payment/views.py

```diff
@@ -241,9 +251,27 @@ def saldo(request):
                     description="Saldo deposit - " + request.user.email
                 )
 
-                PaymentTransaction.objects.create(user=request.user, amount=amount, used_stripe=True)
+                payment_transaction = PaymentTransaction.objects.create(user=request.user, amount=amount,
+                                                                        used_stripe=True)
+
+                receipt = PaymentReceipt(
+                    to_mail=payment_transaction.user.email,
+                    from_mail=settings.EMAIL_TRIKOM,
+                    subject="[kvittering] saldo inskudd",
```

Comment:
Could probably capitalize `[Kvittering]`. Also, it's gramatically correct to write "Saldoinnskudd". 

While you're at it, it could be even more descriptive: "Saldoinnskudd på online.ntnu.no" or something the like.